observatory-platform/observatory/platform/telescopes/workflow_telescope.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Author: Tuan Chien
+
+
+from observatory.platform.telescopes.telescope import Telescope
+
+
+class WorkflowTelescope(Telescope):
+    """
+    Generic Workflow telescope for running tasks.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def make_release(self) -> None:
+        """ Not needed. """
+        return None
```

Comment:
I don't know if it is necessary to make a new class? You might still want to create a Release object when running a workflow, e.g. to store a date or version for the processed dataset.